python/servo/mutation/test.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013 The Servo Project Developers. See the COPYRIGHT
+# file at the top-level directory of this distribution.
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import fileinput
+import re
+import subprocess
+import sys
+import os
+DEVNULL = open(os.devnull, 'wb')
+
+
+def mutate_line(file_name, line_number):
+    for line in fileinput.input(file_name, inplace=True):
+        if fileinput.lineno() == line_number:
+            line = re.sub(r'\s&&\s', ' || ', line)
+        print line.rstrip()
+
+
+def mutation_test(file_name, tests):
+    line_numbers = []
+    for line in fileinput.input(file_name):
+        if re.search(r'\s&&\s', line):
```

Comment:
Again, there's no randomness here. Scaling this up to the whole servo code base is going to require mutating each line with low probability, not trying everything exhaustively.

Reply:
I have added randomness, Now mutation test runs only one mutant per file.

Reply:
Cool!